src/web/security/core/src/main/java/org/geoserver/security/web/url/RegexCheckPage.java

```diff
@@ -90,10 +90,34 @@ private void initUI(IModel<RegexURLCheck> bean) {
         add(form);
     }
 
+    /**
+     * Generate an example reglar expression.
+     *
+     * <p>This example processes {@link #getOWSURL()} to demonstrate:
+     *
+     * <ul>
+     *   <li>Shows {@code https?:} to allow support both http and https
+     *   <li>Escaping {@code \.} characters
+     *   <li>Disallow any {@code ../} relative paths
+     *   <li>Allow anything after {@code ?} query
+     * </ul>
+     *
+     * @return example regular expression
+     */
     private String getDefaultRegex() {
-        return "^" + getOWSURL() + ".*$";
+        String url = getOWSURL();
+
+        url = url.replace("http:", "https?:");
```

Comment:
Disagree... what if a host is only available via http? The default regex should be allowing for both, to give a good example, but don't assume the admin will enter only https urls.

Reply:
You are not reading this correctly, this is ensuring that the default allows both?

I am taking the case where ``http:`` and changing it to ``https?:`` (so the ``s`` is optional).

Reply:
Do you want to also add so both ``http:`` and ``https:`` are covered:
```
url = url.replace("http:", "https?:");
url = url.replace("https:", "https?:");
```

Reply:
I indeed misread it. Still... it's magic. The admin enters a checker that only allows http, and the implementation also allows https. It should be documented, or even better, just be part of the sample regex, and let the admin decide what to do.

Reply:
Removed, this is only an example and we have better things to do.

Reply:
After this change the new default is ``^http://localhost:8080/geoserver/ows((?!\.\./).)*(\?.*)?$``

With normalization handled this can be simplified to ``^http://localhost:8080/geoserver/ows(\?.*)?$``

However I would prefer to simplify to ``^http://localhost:8080/geoserver/ows.*$`` as a better starting point for folks creating their own patterns to lock down external services...

Reply:
Which is basically going full circle and using the original regex?